src/lib/y2firewall/dialogs/main.rb

```diff
@@ -100,6 +100,10 @@ def back_handler
       def fw
         Y2Firewall::Firewalld.instance
       end
+
+      def swap_api
```

Comment:
I would like to see some description here. Why are you assigning a new API object? Why? (I know why, but it would be nice to document it).